doc/oper.xml

```diff
@@ -2257,3 +2289,30 @@ true
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="IsOrderIdeal">
+<ManSection>
+  <Oper Name="IsOrderIdeal" Arg="D, subset" Label="for a digraph and list"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    This function returns <K>true</K> if the specified subset is "downwards" closed, i.e. contains every node less than the given nodes.
```

Comment:
The manual entry should indicate that `D` should satisfy `IsPartialOrderDigraph` and say what happens if `D` does not satisfy `IsPartialOrderDigraph`.